.travis.yml

```diff
@@ -1,24 +1,24 @@
 sudo: required
 language: scala
 scala:
-- 2.12.2
+  - 2.12.2
 jdk:
-- oraclejdk8
+  - oraclejdk8
 script:
-- sbt +clean coverage +test && sbt coverageReport coverageAggregate
-- find $HOME/.sbt -name "*.lock" | xargs rm
-- find $HOME/.ivy2 -name "ivydata-*.properties" | xargs rm
+  - curl -L -o ~/bin/mill https://github.com/lihaoyi/mill/releases/download/0.4.0/0.4.0-9-1db2ee && chmod +x ~/bin/mill
+  - export PATH=~/bin/mill:$PATH
+  - mill __.clean && mill __.test && mill __.scoverage.xmlReport 
 cache:
   directories:
-  - "$HOME/.ivy2/cache"
-  - "$HOME/.sbt/boot/"
+    - "$HOME/.ivy2/cache"
+    - "$HOME/.coursier"
+
 before_install:
-- pip install --user codecov
-- sudo chmod +x /usr/local/bin/sbt
+  - pip install --user codecov
 after_success:
 - codecov
 - test "$TRAVIS_REPO_SLUG" == 'fulcrumgenomics/commons' && test "$TRAVIS_PULL_REQUEST"
-  == 'false' && test "$TRAVIS_BRANCH" == 'master' && sbt +publish
+  == 'false' && test "$TRAVIS_BRANCH" == 'master' && mill mill.scalalib.PublishModule/publishAll "${SONATYPE_USERNAME}:${SONATYPE_PASSWORD}" __.publishArtifacts --release true
```

Comment:
This is a little cumbersome.  Is there a way to put some of this line into the build file so that the command is closer to `mill publishAll`?  Thinking about when we're doing official releases and doing this by hand.

Reply:
Also, is this doing cross-builds?